conference.js

```diff
@@ -1672,6 +1672,7 @@ export default {
             try {
                 await this.localVideo.setEffect(effect);
                 APP.store.dispatch(setVideoMuted(mute, MEDIA_TYPE.PRESENTER));
+                this.setVideoMuteStatus(mute);
```

Comment:
Do you think it would be easy or wise to move this into redux, either middleware or as a state change subscriber?

Reply:
The plan is to move all of presenter mode related code in conference.js into middleware at a later point in time. Can I do it then ?

Reply:
Yeah sounds good.